remoteappmanager/docker/container.py

```diff
@@ -32,12 +32,22 @@ class Container(HasTraits):
     #: the id that will go in the URL of the container
     url_id = Unicode()
 
+    #: The absolute base prefix path for the container.
+    base_urlpath = Unicode(None, allow_none=True)
+
     @property
-    def url(self):
+    def urlpath(self):
         """Returns the relative url of the Container."""
         return "containers/{}".format(self.url_id)
 
     @property
+    def absurlpath(self):
```

Comment:
For the name of `absurlpath`, if we can cope with `base_urlpath`, that would seem that another `abs_urlpath` is okay.

As for where we should compute `urlpath`, I would think a `get_container_url` as a BaseHandler method maybe good place for it.  After all it is web related.